charts/cluster-api-runtime-extensions-nutanix/addons/cluster-autoscaler/values-template.yaml

```
@@ -0,0 +1,36 @@
+fullnameOverride: "cluster-autoscaler-{{ .Cluster.Name }}"
```

This isn't correct any more, see https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/v0.13.7/charts/cluster-api-runtime-extensions-nutanix/templates/cluster-autoscaler/manifests/helm-addon-installation.yaml#L12

```suggestion
fullnameOverride: "cluster-autoscaler-{{ `{{ index .Cluster.Annotations "caren.nutanix.com/cluster-uuid" }}` }}"
```